packages/release-please/src/release-please.ts

```diff
@@ -20,6 +20,10 @@ import { Application } from 'probot';
 // See https://github.com/googleapis/release-please/issues/249
 import { ReleaseType, BuildOptions } from 'release-please/build/src/release-pr';
 import { ReleasePRFactory } from 'release-please/build/src/release-pr-factory';
+import {
+  GitHubRelease,
+  GitHubReleaseOptions,
+} from 'release-please/build/src/github-release';
```

Comment:
Should probably export these types from the index, so consumers are resilient to files moving around.

Reply:
good call.